publicAPI/crest_utils.py

```diff
@@ -294,12 +295,20 @@ def fetch_esi_endpoint(
         'User-Agent': config.get('GLOBAL', 'useragent')
     }
 
-    # no try-except, catch in caller
-    # done to make logging path easier
-    req = requests.get(
-        esi_url,
-        headers=headers
-    )
+    while True:
+        # no try-except, catch in caller
+        # done to make logging path easier
+        req = requests.get(
+            esi_url,
+            headers=headers
+        )
+
+        if req.status_code == 502:      # Server Error: Bad Gateway for url
```

Comment:
Yes, as designed.  I don't want to arrive home 12 hrs later to see that the remote host was down for 2 minutes (or 10) and that caused my data pull to fail.  I want it to keep retrying until *I* tell it to stop.  Of course you're the boss.  :)